packages/azure-services/src/azure-queue/queue.ts

```diff
@@ -84,6 +91,18 @@ export class Queue {
         return queueProperties.approximateMessagesCount;
     }
 
+    private async tryCreateMessage(queue: string, message: unknown): Promise<boolean> {
```

Comment:
this is simple create message func b/c it will throw if fail, can rename to `createMessage`

Reply:
Done.